src/index.html

```diff
@@ -17,11 +17,452 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img
+            src="../src/images/logo.png"
+            alt="logo"
+            class="header__logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="#apple"
+              class="nav__list-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#samsung"
+              class="nav__list-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#smartphones"
+              class="nav__list-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#laptops_and_computers"
+              data-qa="nav-hover"
+              class="nav__list-link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="# gadgets"
```

Comment:
There seems to be an unnecessary space in the `href` attribute value of the anchor tag. It should be `href="#gadgets"` without the space.